src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -92,6 +92,7 @@ public class ComputeEngineCloud extends AbstractCloudImpl {
 
   private transient volatile ComputeClient client;
   private boolean noDelayProvisioning;
+  private boolean jnlpSupport;
```

Comment:
https://github.com/jenkinsci/google-compute-engine-plugin/pull/312#discussion_r2198899107 similarly in the PR title, variable names, etc.